packages/react-server-gulp-module-tagger/index.js

```diff
@@ -22,12 +22,13 @@ module.exports = function(config) {
 	config.basePath = module.filename.replace(THIS_MODULE,'');
 	return forEach(function(stream, file){
 		return stream.pipe(replace(REPLACE_TOKEN, (match, optString) => {
+			const opts = optString ? {label: optString.replace(/.+label:\s?"(\w+)".+/, "$1")} : undefined;
```

Comment:
Couple questions about this regex:
- it looks like it assumes the label in opts will use a double quote, which is true in literally every case I could find, except for one: `PageUtil.js` :)
```
40365: 		var logger = __webpack_require__(303).getLogger({ "label": "{label: 'PageConfig'}", "name": "react-server.core.util.PageUtil.{label: 'PageConfig'}", "color": { "server": 217, "client": "rgb(212,127,127)" } });
```
Seems easy enough to support either quoting style though.

- I'm not sure if we care all that much, since I couldn't find any of these usages, but it looks like with the previous impl, we could write:
```
logger.getLogger(__LOGGER__({
   label: "foo"
}));
```
and it would work correctly. This regex causes invalid syntax to be output in this case, which causes the gulp build to barf. (I tested by adding a multiline options object to PageUtil.js and running `npm run prepublish`)